projects/demo-playwright/tests/addon-table/table-pagination/table-pagination.spec.ts

```diff
@@ -0,0 +1,73 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, Locator, Page, test} from '@playwright/test';
+
+test.describe(`TablePagination`, () => {
+    test.describe(`Dropdown with \`[size]\`-options (amount items per page)`, () => {
+        test(`Basic case`, async ({page}) => {
+            await tuiGoto(page, `/components/table-pagination/API`);
+            const {apiPageExample} = new TuiDocumentationPagePO(page);
+
+            const {tablePagination, linesPerPageSelect} =
+                await initializeAliases(apiPageExample);
+
+            await linesPerPageSelect.click();
+
+            await waitForCheckmarkIcon(page);
+            await expect(tablePagination).toHaveScreenshot(`0-[size]-dropdown-base.png`);
+        });
+
+        test(`With very long option name`, async ({page}) => {
+            const longNumber = Number.MAX_SAFE_INTEGER;
+
+            await tuiGoto(
+                page,
+                `/components/table-pagination/API?items=[0, ${longNumber}]&size=${longNumber}&total=${longNumber}&page=0`,
+            );
+            const {apiPageExample} = new TuiDocumentationPagePO(page);
+            const {tablePagination, linesPerPageSelect} =
+                await initializeAliases(apiPageExample);
+
+            await linesPerPageSelect.click();
+
+            await waitForCheckmarkIcon(page);
+            await expect(tablePagination).toHaveScreenshot(`1-[size]-dropdown-base.png`);
+        });
+    });
+
+    test(`Custom size-option content`, async ({page}) => {
+        await tuiGoto(page, `/components/table-pagination/API`);
```

Comment:
Is it a mistake ? 🤔 

https://github.com/taiga-family/taiga-ui/blob/5d44c31b1fa483a0aa4ebbeb930c8669c0eca917/projects/demo-cypress/cypress/tests/addon-table/table-pagination/table-pagination.cy.ts#L37-L40

https://taiga-ui.dev/components/table-pagination#custom-size-option-content

Reply:
yes, i've updated the test to check against the below. my mistake 😓 

![2--sizeOptionContent--dropdown](https://github.com/taiga-family/taiga-ui/assets/29514264/a010ef3d-2e7e-4ec3-93ed-3670d28a2a8f)